eZ/Publish/API/Repository/Tests/LocationServiceTest.php

```diff
@@ -980,6 +980,41 @@ public function testUpdateLocationStructValues(array $data)
      *
      * @see \eZ\Publish\API\Repository\LocationService::updateLocation()
      * @depends eZ\Publish\API\Repository\Tests\LocationServiceTest::testLoadLocation
+     */
+    public function testUpdateLocationWithSameRemoteId()
+    {
+        $repository = $this->getRepository();
+
+        $locationId = $this->generateId('location', 5);
+        /* BEGIN: Use Case */
+        // $locationId and remote ID is the IDs of the same, existing location
+        $locationService = $repository->getLocationService();
+
+        $originalLocation = $locationService->loadLocation($locationId);
+
+        $updateStruct = $locationService->newLocationUpdateStruct();
+
+        // Remote ID of an existing location with the same locationId
+        $updateStruct->remoteId = '3f6d92f8044aed134f32153517850f5a';
```

Comment:
Do we want to read that value from `$originalLocation` above instead of hardcoding ?